capstone/src/main/java/com/google/sps/servlets/student/StudentServlet.java

```diff
@@ -110,6 +108,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     } else {
       response.sendRedirect("/profile.html");
     }
+
+    // Update student information with edited content
+    String newGradYear = request.getParameter("new-year");
+    String newMajor = request.getParameter("new-major");
+    String newName = request.getParameter("new-name");
```

Comment:
Should these parameter names be constants?

Reply:
Yes, just added constants for them. Thanks for pointing this out!